packages/js/product-editor/src/blocks/generic/taxonomy/create-taxonomy-modal.tsx

```diff
@@ -21,7 +21,7 @@ import classNames from 'classnames';
 /**
  * Internal dependencies
  */
-import { Taxonomy } from './types';
+import { Taxonomy } from '../../../types';
```

Comment:
Super simple suggestion:

```suggestion
import type { Taxonomy } from '../../../types';
```